packages/orchestration/src/utils/chainHub.js

```diff
@@ -25,7 +39,7 @@ export const CONNECTIONS_KEY = 'chainConnection';
  * @param {string} chainId2
  */
 export const connectionKey = (chainId1, chainId2) =>
-  JSON.stringify([chainId1, chainId2].sort());
+  [chainId1, chainId2].sort().join(CHAIN_ID_SEPARATOR);
```

Comment:
I think we should assert that neither id contains `CHAIN_ID_SEPARATOR`

Seems like there's a confusion attack risk otherwise.